src/main/java/com/faforever/client/fa/LaunchCommandBuilder.java

```diff
@@ -231,6 +243,16 @@ public List<String> build() {
       command.add(String.valueOf(deviation));
     }
 
+    if (division != null) {
+      command.add("/division");
+      command.add(division);
+    }
+
+    if (StringUtils.hasText(subdivision)) {
```

Comment:
Why is this one not a null check?

Reply:
Also spring StringsUtils is marked more for internal use so better to use Apache StringUtils

Reply:
The grandmaster division has an empty string as subdivision name

Reply:
Got it then I would still switch the library used away from the internal spring one